src/r5py/util/classpath.py

```diff
@@ -36,7 +36,8 @@
     R5_CLASSPATH = str(pathlib.Path(config.CACHE_DIR) / pathlib.Path(R5_JAR_URL).name)
     try:
         with open(R5_CLASSPATH, "rb") as jar:
-            assert hashlib.sha256(jar.read()).hexdigest == R5_JAR_SHA256
```

Comment:
I saw your comments in the issue, but I don't quite understand why following shouldn't work:
```
assert hashlib.sha256(jar.read()).hexdigest() == R5_JAR_SHA256
```
Perhaps [this comment](https://github.com/r5py/r5py/pull/92#issuecomment-1126889001) might explain the weird behavior that you experienced? Not sure, but would be very strange if this would require saving the hash into a variable for it to work.

Reply:
I’d agree with @HTenkanen  that simply adding the parentheses would be the better option (yes, we don’t have to care about garbage collection in Python, but there’s also no reason for introducing another variable - no reuse or anything)